src/test/regress/expected/multi_sql_function.out

```diff
@@ -324,10 +324,11 @@ SELECT (SELECT 1 FROM test_parameterized_sql limit 1) FROM test_parameterized_sq
 ERROR:  cannot perform distributed planning on this query because parameterized queries for SQL functions referencing distributed tables are not supported
 HINT:  Consider using PL/pgSQL functions instead.
 SELECT test_parameterized_sql_function_in_subquery_where(1);
```

Comment:
would be good to add a comment here, sth like
```sql
-- this fails pg <18
-- succeeds in pg18 because of pg18 commit 0dca5d68d
-- where sql functions behave as PL/pgSQL functions and use the cache
```